### ISSUE TITLE
[Suggestion] Ritual of Weeds

### ISSUE THREAD

**Issue by abculatter2** [WayofTime/BloodMagic#368]
First, before you use the ritual, you'd have to make 'blood weed cuttings'. I don't really know what recipe would be best for this, and honestly it would be fine if the ritual just generated these when activated. These weeds will, once placed on top of an opaque block capable of spawning monsters, quickly spread to nearby blocks of the same description. Light will kill them, and they damage players who stand on them. Also, they prevent mob spawning on the block they are on.

The ritual (which would be advanced, probably either equal to or more difficult then the Well of Suffering) will detect Blood Weed within a certain radius, and produce a constant trickle of blood for a nearby altar based on the amount it finds. Sacrifice runes may or may not apply to the blood it produces.

Basically, it's intended as a way to produce blood from a mob farm, without having to deal with all the problems that arise from them. (Not all of which are lag-related, though that is a part of the reasoning for this suggestion) Obviously, one wouldn't be getting mob drops, and it will probably produce less blood then an actual mob farm (unless you guys want it to) so it's not really any better then the well of suffering, except that it doesn't involve mobs.

**Comment by WayofTime** [comment]
That... doesn't really fit the lore.

**Comment by abculatter2** [comment]
Well, the weeds could be replaced by any stationary alchemical abomination, doesn't have to be plant-based. It's more the concept of 'get trickle of blood from relatively safe block' that I'd like to see.

For example, I'm not sure if souls have useable LP according to the lore, but it could be based on having a lot of special soul sand (or just soul sand) in the area, with the logic being that the soulsand entraps souls that happen to be passing by, and sucks them of their (after)life. Could also be either limited to, or have an output buff in the nether.

There could also be some other kind of entrapment mechanic for mob's soul network, which allows you to entrap them in blocks that can be placed around the ritual. This could require killing a lot of mobs (or maybe an alternative could be the above-stated idea of a ritual that entraps souls with soul sand, but outputs entrapped souls instead of LP directly)
